references/video_classification/train.py

```diff
@@ -154,6 +154,11 @@ def main(args):
     train_resize_size = tuple(args.train_resize_size)
     train_crop_size = tuple(args.train_crop_size)
 
+    if args.clip_len == 0:
+        if args.batch_size != 1:
+            raise ValueError("Dynamic clip length is supported only for batch-size 1.")
```

Comment:
Another gotcha. Batching won't work for variable temporal dimension. So this can only used for batch-size = 1